utils/sources.rb

```diff
@@ -84,9 +84,9 @@ def to_s
       puts <<-EOF
       sources.rb [option] path/package/url ..
 
-      -u, --url                                   just emit URLs
-      -m, --mirror http://example.com/some/path   download from mirror
-      -h, --help                                  show this help
+      -u, --url                                     just emit URLs
```

Comment:
With the above changes in the Makefile, this `--url` option isn't really needed anymore, if I'm not mistaken? But I'm fine with keeping it though.